reproduce.sh

```diff
@@ -32,12 +32,14 @@ while [ "$1" != "" ]; do
 done
 
 mkdir -p $CLUSTERFUZZ_CONFIG_DIR
-if [ ! -d $ROOT_DIRECTORY/ENV ] || ([ $additional_deps_args ] && [ ! -d $ROOT_DIRECTORY/local/bin/android-sdk ]); then
+installed_version=$(cat $CLUSTERFUZZ_CONFIG_DIR/version 2>/dev/null || echo -n)
+version=1
+if ! pipenv graph 2>&1 > /dev/null || [ "$installed_version" != "$version" ] || ([ $additional_deps_args ] && [ ! -d $ROOT_DIRECTORY/local/bin/android-sdk ]); then
```

Comment:
maybe put these four into local var so that it explains what they do
e.g.
pipenv_exists 
version_changed
?

Reply:
Good call, this was getting very hard to read. Broke it up into a few sections.